src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1230,6 +1230,11 @@ void runtime::cpu::CPU_ExternalFunction::register_common_passes(
             return false;
 #endif
         }
+        else if (auto conv = as_type<ngraph::op::GroupConvolution>(const_cast<Node*>(&node)))
+        {
+            Strides stride_1{1, 1};
+            return conv->get_data_dilation_strides() == stride_1;
+        }
```

Comment:
https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/runtime/cpu/pass/cpu_assignment.cpp#L145 would be a better check.